tests/guestlog/guestlog.go

```diff
@@ -90,6 +91,39 @@ var _ = Describe("[sig-compute]Guest console log", decorators.SigCompute, func()
 				Entry("without AutoattachSerialConsole but with LogSerialConsole", false, true, false),
 				Entry("without AutoattachSerialConsole and without LogSerialConsole", false, false, false),
 			)
+
+			It("it should exit cleanly when the shutdown is initiated by the guest", func() {
+				By("Starting a VMI")
+				vmi := tests.RunVMIAndExpectLaunch(cirrosVmi, cirrosStartupTimeout)
+
+				By("Finding virt-launcher pod")
+				virtlauncherPod, err := libvmi.GetPodByVirtualMachineInstance(vmi, testsuite.GetTestNamespace(vmi))
+				Expect(err).ToNot(HaveOccurred())
+				Expect(virtlauncherPod.Spec.Containers).To(HaveLen(3))
```

Comment:
I'd remove this line, it's not useful in this context.

Reply:
done